features/step_definitions/_when_steps.rb

```diff
@@ -70,15 +70,15 @@
 
 When /^I move the story named (.+) (up|down) to the (\d+)(?:st|nd|rd|th) position of the sprint named (.+)$/ do |story_subject, direction, position, sprint_name|
   position = position.to_i
-  story = Story.find(:first, conditions: ['subject=?', story_subject])
-  sprint = Sprint.find(:first, conditions: ['name=?', sprint_name])
+  story = Story.find_by(subject: story_subject)
+  sprint = Sprint.find_by(name: sprint_name)
   story.fixed_version = sprint
 
   attributes = story.attributes
   attributes[:prev] = if position == 1
                         ''
                       else
-                        stories = Story.find(:all, conditions: ['fixed_version_id=? AND type_id IN (?)', sprint.id, Story.types], order: 'position ASC')
+                        stories = Story.where(['fixed_version_id=? AND type_id IN (?)', sprint.id, Story.types]).order('position ASC')
```

Comment:
Line is too long. [134/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.